src/interpreter/terminator.rs

```diff
@@ -514,23 +526,18 @@ impl<'a, 'tcx> EvalContext<'a, 'tcx> {
             traits::VtableClosure(vtable_closure) =>
                 Ok((vtable_closure.closure_def_id, vtable_closure.substs.func_substs)),
 
-            traits::VtableFnPointer(_fn_ty) => {
-                let _trait_closure_kind = self.tcx.lang_items.fn_trait_kind(trait_id).unwrap();
-                unimplemented!()
-                // let llfn = trans_fn_pointer_shim(ccx, trait_closure_kind, fn_ty);
-
-                // let method_ty = def_ty(tcx, def_id, substs);
-                // let fn_ptr_ty = match method_ty.sty {
-                //     ty::TyFnDef(_, _, fty) => tcx.mk_ty(ty::TyFnPtr(fty)),
-                //     _ => unreachable!("expected fn item type, found {}",
-                //                       method_ty)
-                // };
-                // Callee::ptr(immediate_rvalue(llfn, fn_ptr_ty))
+            traits::VtableFnPointer(fn_ty) => {
```

Comment:
Rename this `fn_ty` to `vtable_fn_ptr`. (It has type [`VtableFnPointerData`](http://manishearth.github.io/rust-internals-docs/rustc/traits/struct.VtableFnPointerData.html).)